pandas/core/indexes/base.py

```diff
@@ -80,6 +80,26 @@ def _try_get_item(x):
         return x
 
 
+def _make_invalid_op(name):
```

Comment:
consider moving this function to core/ops (and parameterizing so you can use it for index/series)